pkg/storage/mvcc_value.go

```diff
@@ -118,15 +120,34 @@ func (v MVCCValue) String() string {
 // SafeFormat implements the redact.SafeFormatter interface.
 func (v MVCCValue) SafeFormat(w redact.SafePrinter, _ rune) {
 	if v.MVCCValueHeader != (enginepb.MVCCValueHeader{}) {
+		fields := make([]string, 0)
 		w.Printf("{")
 		if !v.LocalTimestamp.IsEmpty() {
-			w.Printf("localTs=%s", v.LocalTimestamp)
+			fields = append(fields, fmt.Sprintf("localTs=%s", v.LocalTimestamp))
 		}
+		if v.ImportEpoch != 0 {
+			fields = append(fields, fmt.Sprintf("importEpoch=%v", v.ImportEpoch))
+		}
+		w.Print(strings.Join(fields, ", "))
 		w.Printf("}")
 	}
 	w.Print(v.Value.PrettyPrint())
 }
 
+// EncodeMVCCValueForExport strips fields from the MVCCValueHeader that
+// should not get exported out of the cluster.
+func EncodeMVCCValueForExport(mvccValue MVCCValue) ([]byte, error) {
+	// Consider a fast path, where only the roachpb.Value gets exported.
+	// Currently, this only occurs if the value was not imported.
+	if mvccValue.ImportEpoch == 0 {
```

Comment:
It is weird to me that we get the whole header or none of it just depending on this one field?

If we're going to condition on this field, it sorta feels like this should be the only field you get, or, more generally, you should only get fields that are in the condition?